packages/ui/example/app/globals.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
i'm wondering if we should intentionally work in an example that doesn't include tailwind - to ensure there's no leaking across from AIO components and the next-app

Reply:
Removed the tailwind exports.